source/postprocess/point_values.cc

```diff
@@ -77,6 +103,10 @@ namespace aspect
                                  current_point_values[p].end());
           Utilities::MPI::sum (v, this->get_mpi_communicator(),
                                v);
+          // Normalize values, in case points were claimed by multiple processors
+          std::transform (v.begin(), v.end(), v.begin(),
+                          std::bind (std::divides<double>(),
+                                     std::placeholders::_1, n_procs));
```

Comment:
You could run this only if `n_procs>1` since it's a no-op otherwise. 

Since we don't require C++11, you need to write this as `std_cxx11:bind` and `std_cxx11::_1` instead.